manifests/init.pp

```diff
@@ -101,11 +101,6 @@
   String $capkgs_version                   = 'present',
   String $pkg_version                      = 'present',
   Integer $agingtolerance                  = 24,
-  Boolean $nosymlinks                      = true,
-  Boolean $inet6glue                       = false,
-  Boolean $nowarnings                      = true,
-  Boolean $noerrors                        = false,
-  Boolean $randomcron                      = true,
   Optional[Stdlib::Httpurl] $http_proxy    = undef,
```

Comment:
did you remove those on purpose? Aren't they used anymore? Then we've to label it backwards-incompatible.

Reply:
not on purpose and are definitly needed. i have no idea how i managed to bumble that in there